core/src/main/java/org/frankframework/pipes/AsyncSenderWithListenerPipe.java

```diff
@@ -236,47 +236,47 @@ public void setCorrelationIDXPath(String string) {
 		correlationIDXPath = string;
 	}
 
-	/** Namespace defintions for correlationIDXPath. Must be in the form of a comma or space separated list of <code>prefix=namespaceUri</code>-definitions */
+	/** Namespace definitions for correlationIDXPath. Must be in the form of a comma or space separated list of <code>prefix=namespaceUri</code> definitions. */
 	public void setCorrelationIDNamespaceDefs(String correlationIDNamespaceDefs) {
 		this.correlationIDNamespaceDefs = correlationIDNamespaceDefs;
 	}
 
-	/** Key of a PipelineSession-variable. If specified, the value of the PipelineSession variable is used as input for the XPathExpression or stylesheet, instead of the current input message */
+	/** Key of a PipelineSession variable. If specified, the value of the PipelineSession variable is used as input for the XPathExpression or stylesheet, instead of the current input message. */
```

Comment:
```suggestion
	/** Key of a pipeline session variable. If specified, the value of the PipelineSession variable is used as input for the XPathExpression or stylesheet, instead of the current input message. */
```